src/stateMachines/services/oid4vciMachineService.ts

```diff
@@ -0,0 +1,191 @@
+import {URL} from 'react-native-url-polyfill';
+import {v4 as uuidv4} from 'uuid';
+import {CompactJWT, VerifiableCredential} from '@veramo/core';
+import {computeEntryHash} from '@veramo/utils';
+import {CredentialResponse, CredentialSupported} from '@sphereon/oid4vci-common';
+import {CorrelationIdentifierEnum, IBasicCredentialLocaleBranding, IBasicIdentity, IContact, IdentityRoleEnum} from '@sphereon/ssi-sdk.data-store';
+import {
+  CredentialMapper,
+  IIssuer,
+  IVerifiableCredential,
+  OriginalVerifiableCredential,
+  W3CVerifiableCredential,
+  WrappedVerifiableCredential,
+} from '@sphereon/ssi-types';
+import OpenId4VcIssuanceProvider, {CredentialFromOffer} from '../../providers/credential/OpenId4VcIssuanceProvider';
+import {addCredentialBranding, selectAppLocaleBranding} from '../../services/brandingService';
+import {verifyCredential} from '../../services/credentialService';
+import {getContacts} from '../../services/contactService';
+import store from '../../store';
+import {storeVerifiableCredential} from '../../store/actions/credential.actions';
+import {addIdentity} from '../../store/actions/contact.actions';
+import {ICredentialTypeSelection, IVerificationResult, MappedCredentialOffer, OID4VCIMachineContext, OID4VCIMachineEventTypes} from '../../types';
+
+export const initiating = async (_ctx: OID4VCIMachineContext, _event: OID4VCIMachineEventTypes): Promise<OpenId4VcIssuanceProvider> => {
+  const {requestData} = _ctx;
+  return OpenId4VcIssuanceProvider.initiationFromUri({uri: requestData.uri});
+};
+
+export const createCredentialSelection = async (
+  _ctx: OID4VCIMachineContext,
+  _event: OID4VCIMachineEventTypes,
```

Comment:
event is not being used, suggest to remove it to keep the service clean of machine knowledge